experiments/temporal_robustness/run.py

```
@@ -0,0 +1,1213 @@
+        shutil.rmtree(workspace_dir)
+    if runtime_dir.exists():
+        shutil.rmtree(runtime_dir)
+    output_dir.mkdir(parents=True, exist_ok=True)
+    roots = _prepare_roots(workspace_dir, runtime_dir, output_dir)
+    identity = _build_container_identity(runtime_dir)
+    assembled = _assemble_workspace(representative, roots, extra_specs=extra_specs)
+    runtime = load_runtime(next(iter(runtimes)))
+    cmd = _build_docker_command(representative, runtime, roots, identity, interactive=True)
+    start = datetime.now(timezone.utc)
+    try:
+        result = subprocess.run(cmd, check=False)
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)